src/components/ContactCard/ContactGroups.jsx

```diff
@@ -10,7 +10,7 @@ import { fullContactPropTypes } from '../ContactPropTypes'
 import ContactGroupManager from '../ContactGroups/ContactGroupManager'
 import withContactsMutations from '../../connections/allContacts'
 import withGroupsMutations from '../../connections/allGroups'
-import { checkIfGroupAlreadyExists } from '../ContactGroups/helpers/groups'
+import { isGroupAlreadyExist } from '../../helpers/groups'
```

Comment:
Nit: doesGroupAlreadyExist is grammatically correct. The Already is a bit superflous to me.